tests/infra/notifications_handlers/test_slack_notification_handler.py

```
@@ -0,0 +1,59 @@
+async def test_ensure_send_method_calls_chat_postMessage_from_client():
+    client = SlackFakeClient(as_success=True)
+    sut = SlackNotificationHandler(token=token, client=client)
+    await sut.send(notification)
+    assert client.chat_postMessage_is_called
+    assert notification.is_sent
+
+
+async def test_when_the_server_responded_with_an_error_the_handler_should_raises_RuntimeError():
+    client = SlackFakeClient(as_success=True, raises_with=("token_error", {}))
+    sut = SlackNotificationHandler(token=token, client=client)
+    with pytest.raises(TargetNotFound) as e:
```

Ruff (0.13.3)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)